tmtools/io.py

```diff
@@ -14,13 +15,52 @@
 import numpy as np
 
 
-def get_structure(fname):
+def get_structure(fname, format="pdb"):
+    """
+    Load and parse a structure file.
+
+    Parameters
+    ----------
+    fname : str
+       Path to the structure file.
```

Comment:
One tiny nit, but it seems like this line (and the ones further down in the docstring) are indented by 3 characters, not 4.

Reply:
You are right! Didn't catch this whenever I was reading this. Thanks for the heads up